README.md

```diff
@@ -146,9 +146,21 @@ There are two contexts about karmada:
 
 The `karmada-apiserver` is the **main kubeconfig** to be used when interacting with karmada control plane, while `karmada-host` is only used for debugging karmada installation with the host cluster. You can check all clusters at any time by running: `kubectl config view`. To switch cluster contexts, run `kubectl config use-context [CONTEXT_NAME]`
 
+### Uninstall karmada control plane
```

Comment:
Those docs should be in `hack/README.md` not in `/README.md`, it will take users more puzzle at getting started.